apps/mobile/app/ReactotronConfig.js

```diff
@@ -0,0 +1,5 @@
+import Reactotron from "reactotron-react-native";
+
+Reactotron.configure() // controls connection & communication settings
+  .useReactNative() // add all built-in react native plugins
+  .connect(); // let's connect!
```

Comment:
Rename this file to ts?
`apps/mobile/app/ReactotronConfig.js` -> `apps/mobile/app/ReactotronConfig.ts`